### ISSUE TITLE
[FEATURE] Add data lines for event and instance title

### ISSUE THREAD

**Issue by tcdahlberg** [SFDO-Community-Sprints/Summit-Events-App#482]
In the Summit Events App REST API feed make event title and instance title two separate data items for easier use of one or the other.

```
[
  {
    title: "Test Event - Secondary Title",
    event_title: "Test Event",
    instance_title: "Secondary Title",
    timezone: "Central Daylight Time (America/Chicago)",
    start: "2023-04-21T08:00:00.000Z",
    sponsors: "",
    longDesc: "",
    locationType: "",
    locationTitle: "The best place in the World",
    locationMapLink: "https://g.page/sociablecider?share",
    locationBuilding: "",
    locationAddress: "123 Evergreen Blvd., St. Paul, MN",
    instanceDesc: "A short description",
    ID: "a06DS000006oMlRYAU",
    feedButtonText: "Register",
    excludeSearchTag: false,
    eventUrl: "https://customization-drive-6989-dev-ed.scratch.my.salesforce-sites.com/SummitEventsRegister?instanceID=a06DS000006oMlRYAU",
    eventClosed: false,
    end: "2023-04-21T15:30:00.000Z",
    description: "This description appears in feed and should be concise",
    className: "eventOpened",
    capacityTotal: 100,
    capacityRemaining: 100
  }
]
```

**Comment by tcdahlberg** [comment]
switched field names to title_event and title_instance so that the line up next to title in feed.

**Comment by tcdahlberg** [comment]
Included in beta release 0.29.0.1

**Comment by tcdahlberg** [comment]
Included in production release 0.29.0.1
